cmd/sgbouncewizard/main.go

```diff
@@ -55,12 +136,125 @@ func ListRules(w http.ResponseWriter, r *http.Request) {
 
 func generateHash(pwd []byte) string {
 	hash, err := bcrypt.GenerateFromPassword(pwd, bcrypt.MinCost)
+	if err != nil {
+		log.Println(err)
+		return "error"
+	}
+	return string(hash)
+}
+
+func createRuleDB(rule *BounceRule) []byte {
+	db, err := sql.Open("mysql", "root:root@tcp(127.0.0.1:3306)/drop_rules")
+	checkErr(err)
+
+	stmt, err := db.Prepare("INSERT INTO bounce_rule(id,response_code,enhanced_code,regex,priority,description,bounce_action) VALUES(?,?,?,?,?,?,?)")
```

Comment:
Instead of `db.Prepare` followed by `db.Query` consider `db.Exec`. Prepared statements are good for when you have the same insert stmt, and are inserting multiple rows.